src/pspm_pp.m

```diff
@@ -62,6 +62,12 @@
 method  = varargin{1};
 fn      = varargin{2};
 channel = varargin{3};
+
+options = pspm_options(options, 'convert_gaze');
```

Comment:
Just to confirm if `convert_gaze` is intentionally used here.

Reply:
This is a typo. @teddychao